tracer/test/Datadog.Trace.Tests/ApiWebRequestFactoryTests.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Datadog.Trace.Tests
 {
+    [CollectionDefinition(nameof(WebRequestCollection), DisableParallelization = true)]
```

Comment:
I'm a noob here, but I would have expect to have [Collection(nameof(WebRequestCollection)] here and on all test classes, no?  It seems that what we do for other tests

Reply:
🤦‍♂️ Yes, it should